modules/aleph/aleph.install

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * @file
+ * Handles requirements, installation and uninstall for the module.
+ */
+
+/**
+ * Implements hook_requirements().
+ */
+function aleph_requirements($phase) {
+  $requirements = array();
+  // Ensure translations don't break at install time.
+  $t = get_t();
+
+  if (!function_exists('simplexml_load_string')) {
+    $requirements['simplexml'] = array(
+      'title' => 'SimpleXML',
+      'description' => $t('The Aleph module requires SimpleXML to function. Please install and/or enable SimpleXML in your PHP configuration.'),
+      'severity' => REQUIREMENT_ERROR,
+    );
+  }
+
+  if ($phase == 'runtime') {
```

Comment:
Works perfectly. Added patch to show provider when installing profile.
https://github.com/rvk-utd/ding2/pull/1/commits/70d7fc82b21ad7fd4aac43fc4faf99396d5b37fd